std/algorithm/sorting.d

```diff
@@ -2626,6 +2626,173 @@ unittest
     }
 }
 
+// Always leaves median in r[2]. Credit goes to Xinok, see e.g.
+// http://stackoverflow.com/questions/11065963/possible-to-partition-five-elements-by-median-with-six-comparisons
+private void medianOf5(alias less, Range)(Range r)
+{
+    assert(r.length == 5);
+    enum : uint { a, b, c, d, e }
```

Comment:
this enum feels strange, especially as there is nothing similar in the next function.